.github/workflows/website.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches:
+      - main
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  id-token: write
+  issues: write
+  discussions: write
+  packages: write
+  pages: write
+  pull-requests: write
+  repository-projects: write
+  security-events: write
+  statuses: write

```

Comment:
Do we need _all_ of these permissions?

Reply:
Not really, I was just frustrated with the build failing due to not having the right permissions.

Reply:
it works, with less...